app/api-v1/controllers/Attachment/__tests__/attachments.test.js

```diff
@@ -1,9 +1,115 @@
 const { expect } = require('chai')
+const { stub } = require('sinon')
+
 const attachmentController = require('../index')
+const { BadRequestError, NotFoundError } = require('../../../../utils/errors')
+const { jsonAttachment, fileAttachment } = require('../__tests__/attachments_fixtures')
+const db = require('../../../../db')
+
+const getAttachment = async (req) => {
+  try {
+    return await attachmentController.get(req)
+  } catch (err) {
+    return err
+  }
+}
+
+describe('Attachment controller', () => {
```

Comment:
I think we could have some better coverage here. We're not really coverring passing multiple accept header entries and the prioritisation that comes of that. That said if both @n3op2 and @nitro-marky are happy we can leave this for another day.